core-bundle/src/Resources/config/controller.yml

```diff
@@ -55,6 +55,18 @@ services:
         tags:
             - controller.service_arguments
 
+    Contao\CoreBundle\Controller\ContentElement\CodeController:
+        tags:
+            - { name: contao.content_element, category: texts }
```

Comment:
I think we‘ve talked about using PHP attributes for these (at least for the category), haven‘t we?

Reply:
Yes, but IMHO we decided to do this in a separate PR for all controllers (including the already existing).

Reply:
see ce70d9f0f397d1e0f0773dce065215f3dd2c4f2d + be1e53ff6ac505bd1eb18b7312d6f3bfd46ca976